src/pages/questions/QuestionDisplay.jsx

```diff
@@ -57,16 +60,55 @@ const getValueTagExamplesURL = (question) => {
   return "";
 };
 
-const QuestionDisplay = ({ question, answerQuestion, resetFilters }) => {
+const getNbOfQuestionForValue = async ({ type, valueTag }) => {
+  const { data: dataFetched } = await robotoff.questions(
+    "random",
+    type,
+    valueTag,
+    null,
+    null,
+    1
+  );
+  return dataFetched.count;
+};
+
+const QuestionDisplay = ({
+  question,
+  answerQuestion,
+  resetFilters,
+  filterState,
+}) => {
   const { t } = useTranslation();
   const valueTagQuestionsURL = getValueTagQuestionsURL(question);
   const valueTagExamplesURL = getValueTagExamplesURL(question);
+  const [nbOfPotentialQuestion, setNbOfPotentialQuestions] =
+    React.useState(null);
 
   React.useEffect(() => {
+    if (filterState.valueTag) {
+      // value is already in the filter so it's a useless information
+      setNbOfPotentialQuestions(null);
+      return;
+    } else {
+      let validRequest = true;
+      getNbOfQuestionForValue({
+        type: question?.insight_type,
+        valueTag: question?.value_tag,
+      }).then((nbQuestions) => {
+        if (validRequest) {
+          setNbOfPotentialQuestions(nbQuestions);
+        }
+      });
+      return () => {
+        validRequest = false;
+      };
+    }
+  }, [filterState.valueTag, question?.insight_type, question?.value_tag]);
 
+  React.useEffect(() => {
     function handleShortCut(event) {
-      const preventShortCut = event.target.tagName.toUpperCase() === 'INPUT'
-      if (question && !preventShortCut) {
+      const preventShortCut = event.target.tagName.toUpperCase() === "INPUT";
+      if (question?.insight_id && !preventShortCut) {
         switch (event.keyCode) {
           case 75:
             answerQuestion({ value: -1, insightId: question.insight_id });
```

Comment:
off topic, but, would be nice to have somewhere an idea of what these values (-1, etc) represent, as just looking at the code one is kind of clueless